packages/victory-native/README.md

```diff
@@ -0,0 +1,98 @@
+# Victory Native
+
+[![Maintenance Status][maintenance-image]](#maintenance-status)
+
+### Usage
+
+Install victory-native:
+```sh
+$ npm install victory-native --save
+```
+
+Install react-native-svg:
+```sh
+$ npm install react-native-svg --save
+```
+
+Link react-native:
+```sh
+$ react-native link react-native-svg
+```
+**`victory-native@^33.0.0` requires `react-native-svg@^9.0.0` and `react-native@~0.60.0`**
+
+**Please see [Peer Dependencies and Version Requirements](#peer-dependencies-and-version-requirements) for requirements for previous versions of `victory-native`**
+
+
+Import charts from `victory-native`. For example,
+
+```jsx
+import React, { Component } from "react";
+
+import { VictoryBar } from "victory-native";
+
+class App extends Component {
+  render() {
+    return (
+      <VictoryBar />
+    );
+  }
+}
+
+export default App;
+```
+
+### Peer Dependencies and Version Requirements
+
+**Note:** `victory-native` requires the following peer dependencies:
+- `react-native-svg`
+- `react`
+- `react-native`
+
+**Note:** `react-native-svg` has strict version requirements for both `react` and `react-native`. Please match versions to those required by `react-native-svg`. See the up-to-date requirements on the [react-native-svg Readme][react-native-svg-readme].
+We encourage you to use the latest version of `react-native-svg` possible for your project, as `victory-native` issues are frequently solved by `react-native-svg` bugfixes.
+
+- `victory-native@^33.0.0` requires `react-native-svg@^9.0.0` and `react-native@~0.60.0`
+* `victory-native@^30.0.0` requires `react-native-svg@6.1.x`  or `react-native-svg@^6.5.0`and above
+* `victory-native@^0.16.2` requires `react-native-svg@6.1.x` or `react-native-svg@^6.5.0`
+* ~~`victory-native@~0.16.0` requires `react-native-svg@6.0.0`~~ No longer supported
+* ~~`victory-native@~0.15.0` requires `react-native-svg@^5.0.0`~~ No longer supported
+
+### Local Development and Demo
+
+If you'd like to contribute to `victory-native`, you can use the local demo app to test your changes on the iOS simulator.
+
+```sh
+# Install victory and its dependencies
+$ git clone https://github.com/FormidableLabs/victory
+$ cd victory
+$ yarn install
+# Open up the React Native demo app
+$ cd demo/rn
+$ yarn install
+$ yarn start
+
+# Run the demo from a new terminal window
+$ cd demo/rn && npx react-native run-ios
+```
+
+Changes to the Victory and Victory Native source code will be reflected in the demo app.
```

Comment:
This is effectively the only thing I changed in this file, which just boils down to changing demo location and install instructions.

Reply:
MINOR: Is `react-native` otherwise available in deps already from another path? A lot of the times `npx` can add a lot of unneeded install time for big packages that are already installed nearby....

Reply:
Oh hey, that's a great point! And since we moved from Vanilla RN to Expo, this command is outdated anyways. Updating now!